### PR TITLE
Add toolchain sources to cgmanifest.json

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Update cgmanifest.json with source versions that are used in toolchain build process.

###### Change Log  

- Update cgmanifest

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- N/A

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -3200,6 +3380,266 @@
...
+          "version": "1.6.5",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libX11-1.6.5.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libxau",
```
Comment (christopherco) [inline cgmanifest.json:3397]: does this need to be `libXau`?

**Review by christopherco — APPROVED** [review]

File: cgmanifest.json [hunk 2]
```diff
@@ -3200,6 +3380,266 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libX11",
```
Comment (PawelWMS) [inline cgmanifest.json:3387]: Aren't most `libX*` packages in the CBL-MarinerCoreUI repo and not here? Also, I think these aren't part of the toolchain, right? The CBL-MarinerCoreUI repo has got its own `cgmanifest.json` btw.

File: cgmanifest.json [hunk 3]
```diff
@@ -3200,6 +3380,266 @@
...
+          "version": "1.1.4",
+          "downloadUrl": "https://www.x.org/pub/individual/lib/libXxf86dga-1.1.4.tar.bz2"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libXxf86vm",
```
Comment (PawelWMS) [inline cgmanifest.json:3637]: The `libxcrypt` right after this one is no longer in alphabetical order.

File: cgmanifest.json [hunk 4]
```diff
@@ -3874,249 +4314,409 @@
...
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nss-altfiles",
-          "version": "2.23.0",
-          "downloadUrl": "https://github.com/aperezdc/nss-altfiles/archive/v2.23.0.tar.gz"
+          "name": "openjdk8",
+          "version": "1.8.0.212",
```
Comment (PawelWMS) [inline cgmanifest.json:4568]: I think this one should go before the version above.

File: cgmanifest.json [hunk 5]
```diff
@@ -3874,249 +4314,409 @@
...
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nvidia-modprobe",
-          "version": "450.57",
-          "downloadUrl": "https://github.com/NVIDIA/nvidia-modprobe/archive/450.57.tar.gz"
+          "name": "openjdk8_aarch64",
+          "version": "1.8.0.292",
```
Comment (PawelWMS) [inline cgmanifest.json:4648]: Similarly here - older versions first.

File: cgmanifest.json [hunk 6]
```diff
@@ -6020,6 +6640,36 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "recordproto",
```
Comment (PawelWMS) [inline cgmanifest.json:6647]: The `redis` package above should be behind this one.

File: cgmanifest.json (hunk 2 above)
Comment (anphel31) [reply cgmanifest.json:3387]: Yes, I think a number of these libX* packages are also in CoreUI. However, they are downloaded as part of the toolchain build process in Core, and are required to compile OpenJDK from source.

File: cgmanifest.json (hunk 3 above)
Comment (anphel31) [reply cgmanifest.json:3647]: thanks, fixed

File: cgmanifest.json (hunk 4 above)
Comment (anphel31) [reply cgmanifest.json:4558]: correct - fixed

File: cgmanifest.json (hunk 5 above)
Comment (anphel31) [reply cgmanifest.json:4638]: fixed

File: cgmanifest.json (hunk 6 above)
Comment (anphel31) [reply cgmanifest.json]: fixed

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**